index.js

```diff
@@ -13,6 +13,9 @@ import { createListing } from './src/js/listeners/post/createListing.js';
 import { showListings } from './src/js/views/admin/showListings.js';
 import { searchListings, filterListings } from './src/js/listeners/post/searchListing.js'
 import '/src/scss/index.scss';
+import { adjustH2FontSize } from './listingTemplate.js';
```

Comment:
Your file import here seems to be incorrect and is resulting in a linting error. Please take a look at this